polygon/sync/block_downloader.go

```diff
@@ -162,7 +166,13 @@ func (d *blockDownloader) downloadBlocksUsingWaypoints(ctx context.Context, wayp
 				"kind", reflect.TypeOf(waypointsBatch[0]),
 				"peerCount", len(peers),
 				"maxWorkers", d.maxWorkers,
+				"blk/s", fmt.Sprintf("%.2f", float64(blockCount)/time.Since(fetchStartTime).Seconds()),
+				"bytes/s", fmt.Sprintf("%s", common.ByteCount(uint64(float64(blocksTotalSize)/time.Since(fetchStartTime).Seconds()))),
 			)
+
+			blockCount = 0
+			blocksTotalSize = 0
```

Comment:
are we not meant to reset `fetchStartTime := time.Now()` here too? I know we went back and forth with this but if we are going down the route of showing the bytes/s block/s per interval then we should reset the time counter here too?